index.html

```diff
@@ -685,6 +719,27 @@ <h3>Converting between <code>tiltX</code> / <code>tiltY</code> and <code>altitud
 </pre>
             </section>
         </section>
+        <section>
+            <h2><code>DeviceProperties</code> interface</h2>
+            <pre class="idl">
+                dictionary DevicePropertiesInit {
+                    long uniqueId = -1;
+                };
+
+                [Exposed=Window]
+                interface DeviceProperties {
+                    constructor(optional DevicePropertiesInit devicePropertiesInitDict = {});
+                    readonly attribute long uniqueId;
+                };
+            </pre>
+            <dl data-dfn-for="DeviceProperties" data-link-for="DeviceProperties">
+                <dt><dfn>uniqueId</dfn></dt>
+                    <dd>
+                        <p>A unique identifier for the pointing device. If the hardware supports multiple pointers, pointer events generated from pointing devices must only get a <code>uniqueId</code> if those pointers are uniquely identifiable over the session. If the pointer is uniquely identifiable, the assigned <code>uniqueId</code> to that pointing device will remain constant for the remainder of the session. The <code>uniqueId</code> value of <code>-1</code> MUST be reserved and used to indicate events whose generating device could not be identified. Like <a href="#dom-pointerevent-pointerid">pointerId</a>, to minimize the chance of fingerprinting and tracking across different pages or domains, the <code>uniqueId</code> MUST only be associated explicitly with that particular pointing device for the lifetime of the page / session, and a new randomized <code>uniqueId</code> MUST be chosen the next time that particular pointing device is used again in a new session.</p>
+                        <div class="note">Due to digitizer and pointing device hardware constraints, a <code>uniqueId</code> is not guaranteed to be available for all pointer events from a pointing device. For example, the device may not report its hardware id to the digitizer in time for <code>pointerdown</code> to have a <code>uniqueId</code>. In such a case, the <code>uniqueId</code> may initially be <code>-1</code> and change to a valid value.</div>
```

Comment:
Suggesting a minor change in the last part to avoid a possible misinterpretation that reporting a `-1` before _every_ `pointerdown` is okay:

> For example, the device may not report its hardware id to the digitizer in time before the first <code>pointerdown</code>. In such a case, the <code>uniqueId</code>s for the events from the device will initially be reported as <code>-1</code> before they switch to a valid constant value for the remainder of the session.

Reply:
A bit of an issue here - we know for a fact that older models of surface pen don't report their device id to the digitizer at the beginning of _each_ stroke :( . In this case there is a -1 before every pointer down. What do you think of this?

Reply:
Hmm, that's tricky!  Perhaps the spec should say this?
> The deviceid MAYBE zero when the pointer is not in "active button state".

Additionally, this calls for a clarification to "the remainder of the session" here.  Does the id remain constant only between `pointerdown` and `pointerup` then changes (or may change) at next `pointerdown`?  Or the same constant is reported between all pairs of `pointerdown` and `pointerup` from the same device?

Reply:
>In this case there is a -1 before every pointer down.?
I should clarify that this includes `pointerdown` as well. Unfortunately, it's just a reality of some hardware.